content/20.pharmaceuticals.md

```diff
@@ -273,12 +273,13 @@ For COVID-19, the NSP5 in SARS-CoV-2 contains a cysteine at the active site of M
 Interestingly there has been some argument that selenium deficiency may be associated with more severe COVID-19 outcomes [@doi:10.3390/nu12072098; @doi:10.1017/S0007114520003128; @doi:10.1016/j.redox.2020.101715], possibly indicating that its antioxidative properties are protective [@doi:10.1016/j.freeradbiomed.2020.06.032].
 On the other hand, ebselen and the other compounds identified are likely to be promiscuous binders, which could diminish their therapeutic potential [@doi:10.1038/s41586-020-2223-y].
 While there is clear computational and _in vitro_ support for ebselen's potential as a COVID-19 therapeutic, results from clinical trials are not yet available for this compound.
+However, as of July 2020, phase II clinical trials commenced to assess the effects of SPI-1005, an investigational drug from Sound Pharmaceuticals that contains ebselen [@url:https://www.contractpharma.com/contents/view_breaking-news/2020-08-31/fda-clears-spis-ebselen-for-phase-ii-covid-19-trials/], on 60 adults presenting with each of moderate [@clinicaltrials:NCT04484025] and severe [@clinicaltrials:NCT04483973] COVID-19.
 
 In summary, N3 is a computationally designed molecule that inhibits the viral transcription through inhibiting Mpro.
 Ebselen is both a strong Mpro inhibitor and strong inhibitor of viral replication _in vitro_ that was found to reduce SARS-CoV-2 viral load even more effectively than N3.
 Ebselen is a very promising compound since its safety has been demonstrated in other indications.
-However, Ebselen is likely a false positive since it is a promiscuous compound that can have many targets [@doi:10.1039/c8cc04258f].
-Therefore, compounds with higher specificity may be required to effectively translate to clinical trials.
+However, Ebselen may be a false positive, since it is a promiscuous compound that can have many targets [@doi:10.1039/c8cc04258f].
```

Comment:
```suggestion
However, ebselen may be a false positive, since it is a promiscuous compound that can have many targets [@doi:10.1039/c8cc04258f].
```